setup/install/storage.md

```diff
@@ -20,4 +20,7 @@ Spinnaker, you must configure it to use one of the supported storage types.
 
 ## Next Steps
 
-Now we're ready to [deploy Spinnaker](/setup/install/deploy).
+This at this point we've done all the configuration required to get Spinnaker
+to start up, but Spinnaker is not in a very functional state. So let's see [what
```

Comment:
This is still awkward. Especially because storage is required but not other things, like security or even accounts. Spinnaker has no practical value without an account so is just as useless as if it had no storage. And once you add accounts, you want to consider security. Storage being intrinsically required by the infrastructure as opposed to the application is a special case. Either it should be treated differently, or you should just configure everything first, then say you can add and change things later.